wire-ios/Wire-iOS/Sources/Helpers/SemanticColors.swift

```diff
@@ -55,6 +55,8 @@ public enum SemanticColors {
         static let textTabBar = UIColor(light: Asset.Colors.gray70, dark: Asset.Colors.gray60)
         static let textFieldFloatingLabel = UIColor(light: Asset.Colors.gray80, dark: Asset.Colors.gray50)
         static let textSecurityEnabled = UIColor(light: Asset.Colors.green500Light, dark: Asset.Colors.green500Dark)
+        static let textInactive = UIColor(light: Asset.Colors.gray60, dark: Asset.Colors.gray70)
```

Comment:
Can we rename this property to something more descriptive? textInactive isn't clear enough to where it's being used.

Reply:
Can you suggest a name @agisilaos ?

Reply:
@johnxnguyen How about `textParticipantConnecting`?

Reply:
I would argue that `textInactive` is better. It suggests that it will be used for labels that are inactive. Maybe 
`textDimmed` could also be good. 

Being more descriptive also makes it more restrictive because now this color could only be used for a single purpose (for connecting participants). In my opinion that isn't semantic coloring (well defined semantics that are reusable), it's just naming colors (single purpose colors).

Reply:
Sounds good! I think it's fair enough reasoning to keep it as it is for now. Good to go from my side. 👍